src/test/java/com/bobocode/hoverla/bring/context/BeanInitializerTest.java

```diff
@@ -0,0 +1,63 @@
+package com.bobocode.hoverla.bring.context;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Stream;
+
+import static java.util.Collections.emptyList;
+import static java.util.Collections.singletonList;
+import static org.mockito.Mockito.*;
+
```

Comment:
I thought we'd need `@ExtendWith(MockitoExtension.class)` to make it work and be able to use mocks. How does it work without this extension?